ui/app/src/models/ContentType.ts

```diff
@@ -35,6 +35,8 @@ export interface ContentTypeField {
   defaultValue: string
   required: boolean;
   fields?: LookupTable<ContentTypeField>;
+  values?: LookupTable<string>[];
```

Comment:
{ label: string, value: string }[]

Reply:
values here is an object with the configuration fields, like 
`
{
   id: 'segment',
   type: 'dropdown',
   defaultValue: 'test',
   ...
}
`